main.go

```diff
@@ -87,13 +83,21 @@ func main() {
 	SETTINGS.Set("prometheus-monitoring", "n", "add promethues monitoring endpoint")
 	SETTINGS.Set("STORAGEMETHOD", "bytes", "Storagemethod available options are json, jsonz, bytes, bytesz")
 	SETTINGS.Set("LOADATSTARTUP", "n", "Load data at startup. ('y', 'n')")
-	SETTINGS.Parse()
 
-	//Construct yes or no to booleans in SETTINGS
+	SETTINGS.Set("readonly", "yes", "only allow read only funcions")
+	SETTINGS.Set("debug", "no", "print memory usage")
+
+	SETTINGS.Set("groupbycache", "yes", "use in memory cache")
 
-	ITEMS = make(Items, 0, 100*1000)
```

Comment:
I think I moved all Items code to store.go.  Also inital size of Items leaves a lot of nil values when doing append. Since new items are added behind. if i rember correctly maybe some tests are needed :)